paper/content/chapter2/chapter2_1.tex

```diff
@@ -1,9 +1,156 @@
 % !TeX root = ../../paper.tex
-\subsection{Evaluierungsfunktion}
+\subsection{Bewertungsfunktion}
 
+In der Spieltheorie ist es in der Regel nicht möglich, alle möglichen Zugfolgen aus einer Spielposition heraus bis zum Ende zu verfolgen.
```

Comment:
(In der Spieltheorie) ->Beim Schach

Reply:
Die Quelle bezieht sich allerdings auf die Spieltheorie im Allgemeinen (was in diesem Fall auf Schach im Besonderen auch auftritt).

Reply:
Okay dann lass es so.